### PR TITLE
Triton model support batching

### PR DESCRIPTION
**Pull request by junjiejiangjjj**
Signed-off-by: junjie.jiang <junjie.jiang@zilliz.com>

### CODE REVIEW

File: towhee/serve/triton/bls/pipeline_model.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def auto_complete_config(auto_complete_model_config):
         input0 = {'name': 'INPUT0', 'data_type': 'TYPE_STRING', 'dims': [1]}
         output0 = {'name': 'OUTPUT0', 'data_type': 'TYPE_STRING', 'dims': [1]}
 
-        auto_complete_model_config.set_max_batch_size(0)
+        auto_complete_model_config.set_max_batch_size(8)
```
Comment (GuoRentong) [inline towhee/serve/triton/bls/pipeline_model.py:30]: why is the magic number 8?

File: towhee/runtime/runtime_pipeline.py [hunk 2]
```diff
@@ -89,23 +90,27 @@ def result(self) -> any:
         self.time_profiler.record(Event.pipe_name, Event.pipe_out)
         return res
 
-    def async_call(self, inputs: Tuple):
+    def async_call(self, inputs: Union[Tuple, List]):
         self.time_profiler.inputs = inputs
-        self._data_queues[0].put(inputs)
-        self._data_queues[0].seal()
+        self._input_q.put(inputs)
```
Comment (GuoRentong) [inline towhee/runtime/runtime_pipeline.py:95]: use _input_queue instead

**Review by GuoRentong — CHANGES_REQUESTED** [review]

File: towhee/serve/triton/bls/pipeline_model.py (hunk 1 above)
Comment (junjiejiangjjj) [reply towhee/serve/triton/bls/pipeline_model.py:30]: Can achieve a better performance in image-pipeline, and will be configured in the next version.

**Pull request merged**